src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  background-color: #fff;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)